endpoint.bal

```diff
@@ -129,16 +136,18 @@ public client class Client {
     # + queueResourcePath - Resource path to the queue from the host address. e.g.: /610968236798/myQueue.fifo 
     # + attributes - Non-mandatory parameters for receiving a message
     # + return - If success, details of the received message, else returns error
-    public remote function receiveMessage(string queueResourcePath, map<string> attributes) 
-            returns @tainted InboundMessage[]|OperationError {
-        string amzTarget = AMAZON_SQS_API_VERSION + "." + ACTION_RECEIVE_MESSAGE;
+    @display {label: "Receive message in queue"}
+    remote isolated function receiveMessage(@display {label: "Resource path to queue"} string queueResourcePath, 
```

Comment:
Please check my above comment. You have to follow up the API Reference doc and implement all parameters in the receiveMessage operation.

Reply:
Noted and Resolved